<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>441</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Frequencia de pulsos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo em alta dos pulsos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo em baixa dos pulsos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Frequencia de burst</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo em alta do burst</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo em baixa do burst</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo de subida</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo de plato</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo de descida</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tempo de repouso</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Amplitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Padrao de pulso</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>I2cAddr:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAddr">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOpen">
         <property name="text">
          <string>OpenComm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>CloseComm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
